2024-07-15 18:12:35,622 (7b2e8d6ee808) :  INFO (core:349) - Starting framework core - Version: 2.6.3, Build: 4245626 (Tue Jul 09 11:55:06 UTC 2024)
2024-07-15 18:12:35,622 (7b2e8d6ee808) :  DEBUG (core:361) - Using the elevated policy
2024-07-15 18:12:35,623 (7b2e8d6ee808) :  DEBUG (core:450) - Starting runtime component.
2024-07-15 18:12:35,625 (7b2e8d6ee808) :  DEBUG (core:450) - Starting caching component.
2024-07-15 18:12:35,625 (7b2e8d6ee808) :  DEBUG (core:450) - Starting data component.
2024-07-15 18:12:35,626 (7b2e8d6ee808) :  DEBUG (core:450) - Starting networking component.
2024-07-15 18:12:35,626 (7b2e8d6ee808) :  DEBUG (networking:266) - No cookie jar found
2024-07-15 18:12:35,627 (7b2e8d6ee808) :  DEBUG (networking:434) - Setting the default network timeout to 20.0
2024-07-15 18:12:35,629 (7b2e8d6ee808) :  DEBUG (core:450) - Starting localization component.
2024-07-15 18:12:35,629 (7b2e8d6ee808) :  INFO (localization:409) - Setting the default locale to en-us
2024-07-15 18:12:35,629 (7b2e8d6ee808) :  DEBUG (core:450) - Starting messaging component.
2024-07-15 18:12:35,630 (7b2e8a2a8b38) :  DEBUG (networking:144) - Requesting 'http://127.0.0.1:32400/system/messaging/clear_events/com.plexapp.agents.thetvdb'
2024-07-15 18:12:35,632 (7b2e8d6ee808) :  DEBUG (core:450) - Starting debugging component.
2024-07-15 18:12:35,633 (7b2e8d6ee808) :  DEBUG (core:450) - Starting services component.
2024-07-15 18:12:35,633 (7b2e8d6ee808) :  DEBUG (core:450) - Starting myplex component.
2024-07-15 18:12:35,633 (7b2e8d6ee808) :  DEBUG (core:450) - Starting notifications component.
2024-07-15 18:12:35,831 (7b2e8d6ee808) :  DEBUG (accessor:68) - Creating a new model access point for provider com.plexapp.agents.thetvdb in namespace 'metadata'
2024-07-15 18:12:35,837 (7b2e8d6ee808) :  DEBUG (networking:144) - Requesting 'http://127.0.0.1:32400/:/plugins/com.plexapp.system/resourceHashes'
2024-07-15 18:12:35,850 (7b2e8d6ee808) :  ERROR (networking:197) - Error opening URL 'http://127.0.0.1:32400/:/plugins/com.plexapp.system/resourceHashes'
2024-07-15 18:12:35,853 (7b2e8d6ee808) :  CRITICAL (runtime:1299) - Exception getting hosted resource hashes (most recent call last):
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-424562606/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/runtime.py", line 1293, in get_resource_hashes
    json = self._core.networking.http_request("http://127.0.0.1:32400/:/plugins/com.plexapp.system/resourceHashes", timeout=10).content
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-424562606/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/networking.py", line 243, in content
    return self.__str__()
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-424562606/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/networking.py", line 221, in __str__
    self.load()
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-424562606/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/networking.py", line 159, in load
    f = self._opener.open(req, timeout=self._timeout)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 435, in open
    response = meth(req, response)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 548, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 473, in error
    return self._call_chain(*args)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 407, in _call_chain
    result = func(*args)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 556, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
HTTPError: HTTP Error 404: Not Found

2024-07-15 18:12:35,863 (7b2e8d6ee808) :  DEBUG (runtime:1117) - Created a thread named 'load_all_services'
2024-07-15 18:12:35,863 (7b2e8a09ab38) :  DEBUG (services:265) - Plug-in is not daemonized - loading services from system
2024-07-15 18:12:35,864 (7b2e8d6ee808) :  DEBUG (runtime:1117) - Created a thread named 'get_server_info'
2024-07-15 18:12:35,865 (7b2e8d6ee808) :  DEBUG (core:150) - Finished starting framework core
2024-07-15 18:12:35,865 (7b2e8d6ee808) :  DEBUG (core:560) - Loading plug-in code
2024-07-15 18:12:35,909 (7b2e8a054b38) :  DEBUG (networking:144) - Requesting 'http://127.0.0.1:32400'
2024-07-15 18:12:35,916 (7b2e8a09ab38) :  DEBUG (networking:144) - Requesting 'http://127.0.0.1:32400/:/plugins/com.plexapp.system/messaging/function/X0J1bmRsZVNlcnZpY2U6QWxsU2VydmljZXM_/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMApyMAo_'
2024-07-15 18:12:35,950 (7b2e8a054b38) :  DEBUG (core:538) - Machine identifier is 9d8693d951a7aeb9fa19783ceff100b02030f820
2024-07-15 18:12:35,957 (7b2e8a054b38) :  DEBUG (core:539) - Server version is 1.40.4.8679-424562606
2024-07-15 18:12:36,280 (7b2e8a09ab38) :  DEBUG (services:362) - Loaded services
2024-07-15 18:12:36,311 (7b2e8a077b38) :  DEBUG (services:438) - No shared code to load
2024-07-15 18:12:36,421 (7b2e8d6ee808) :  DEBUG (core:566) - Finished loading plug-in code
2024-07-15 18:12:36,422 (7b2e8d6ee808) :  DEBUG (agentkit:1132) - Creating new agent class called TVDBAgent
2024-07-15 18:12:36,423 (7b2e8d6ee808) :  DEBUG (agentkit:937) - Updating agent information: [{'media_types': ['TV_Show'], 'accepts_from': None, 'fallback_agent': None, 'contributes_to': None, 'languages': ['en', 'fr', 'zh', 'sv', 'no', 'da', 'fi', 'nl', 'de', 'it', 'es', 'pl', 'hu', 'el', 'tr', 'ru', 'he', 'ja', 'pt', 'cs', 'ko', 'sl', 'hr'], 'persist_stored_files': True, 'version': 0, 'primary_provider': True, 'prefs': True, 'name': 'TheTVDB'}]
2024-07-15 18:12:36,424 (7b2e8d6ee808) :  DEBUG (networking:144) - Requesting 'http://127.0.0.1:32400/:/plugins/com.plexapp.system/messaging/function/X0FnZW50U2VydmljZTpVcGRhdGVJbmZv/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQo1CmRpY3QKbGlzdApkaWN0Cmxpc3QKbGlzdAoyCnMyNgpjb20ucGxleGFwcC5hZ2VudHMudGhldHZkYnMxMAppZGVudGlmaWVycjEKczEwCmFnZW50X2luZm8xCnIyCjEwCnIzCnMxMQptZWRpYV90eXBlc25zMTIKYWNjZXB0c19mcm9tbnMxNApmYWxsYmFja19hZ2VudG5zMTQKY29udHJpYnV0ZXNfdG9yNApzOQpsYW5ndWFnZXNiMXMyMApwZXJzaXN0X3N0b3JlZF9maWxlc2kwCnM3CnZlcnNpb25iMXMxNgpwcmltYXJ5X3Byb3ZpZGVyYjFzNQpwcmVmc3M3ClRoZVRWREJzNApuYW1lMQpzNwpUVl9TaG93MjMKczIKZW5zMgpmcnMyCnpoczIKc3ZzMgpub3MyCmRhczIKZmlzMgpubHMyCmRlczIKaXRzMgplc3MyCnBsczIKaHVzMgplbHMyCnRyczIKcnVzMgpoZXMyCmphczIKcHRzMgpjc3MyCmtvczIKc2xzMgpocnIwCg__'
2024-07-15 18:12:54,641 (7b2e8d6ee808) :  INFO (core:611) - Started plug-in
2024-07-15 18:12:54,641 (7b2e8d6ee808) :  DEBUG (socketinterface:160) - Starting socket server
2024-07-15 18:12:54,642 (7b2e8d6ee808) :  DEBUG (runtime:1117) - Created a thread named 'start'
2024-07-15 18:12:54,643 (7b2e8d6ee808) :  INFO (socketinterface:184) - Socket server started on port 38543
2024-07-15 18:12:54,643 (7b2e8d6ee808) :  INFO (pipeinterface:25) - Entering run loop
2024-07-15 18:12:54,643 (7b2e8d6ee808) :  DEBUG (runtime:717) - Handling request GET /:/prefixes
2024-07-15 18:12:54,646 (7b2e8d6ee808) :  DEBUG (runtime:814) - Found route matching /:/prefixes
2024-07-15 18:12:54,647 (7b2e8d6ee808) :  DEBUG (runtime:924) - Response: [200] MediaContainer, 151 bytes
2024-07-15 18:12:54,669 (7b2e8961eb38) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.thetvdb
2024-07-15 18:12:54,673 (7b2e895f7b38) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.thetvdb
2024-07-15 18:12:54,677 (7b2e895d4b38) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.thetvdb
2024-07-15 18:12:54,706 (7b2e8961eb38) :  CRITICAL (runtime:493) - Exception matching route for path "/" (most recent call last):
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-424562606/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/runtime.py", line 464, in match_route
    raise Framework.exceptions.FrameworkException("No route found matching '%s'" % path)
FrameworkException: No route found matching '/'

2024-07-15 18:12:54,708 (7b2e8961eb38) :  ERROR (runtime:846) - Could not find route matching /:/plugins/com.plexapp.agents.thetvdb
2024-07-15 18:12:54,709 (7b2e895f7b38) :  CRITICAL (runtime:493) - Exception matching route for path "/" (most recent call last):
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-424562606/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/runtime.py", line 464, in match_route
    raise Framework.exceptions.FrameworkException("No route found matching '%s'" % path)
FrameworkException: No route found matching '/'

2024-07-15 18:12:54,711 (7b2e895f7b38) :  ERROR (runtime:846) - Could not find route matching /:/plugins/com.plexapp.agents.thetvdb
2024-07-15 18:12:54,710 (7b2e8961eb38) :  DEBUG (runtime:88) - Sending packed state data (107 bytes)
2024-07-15 18:12:54,711 (7b2e895d4b38) :  CRITICAL (runtime:493) - Exception matching route for path "/" (most recent call last):
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-424562606/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/runtime.py", line 464, in match_route
    raise Framework.exceptions.FrameworkException("No route found matching '%s'" % path)
FrameworkException: No route found matching '/'

2024-07-15 18:12:54,713 (7b2e895d4b38) :  ERROR (runtime:846) - Could not find route matching /:/plugins/com.plexapp.agents.thetvdb
2024-07-15 18:12:54,712 (7b2e8961eb38) :  DEBUG (runtime:924) - Response: [404] NoneType, 0 bytes
2024-07-15 18:12:54,712 (7b2e895f7b38) :  DEBUG (runtime:88) - Sending packed state data (107 bytes)
2024-07-15 18:12:54,714 (7b2e895d4b38) :  DEBUG (runtime:88) - Sending packed state data (107 bytes)
2024-07-15 18:12:54,715 (7b2e895d4b38) :  DEBUG (runtime:924) - Response: [404] NoneType, 0 bytes
2024-07-15 18:12:54,715 (7b2e895f7b38) :  DEBUG (runtime:924) - Response: [404] NoneType, 0 bytes
